### PR TITLE
Consistant alignment, colors, and font sizes for ngx-select and ngx-inputs

### PR DESCRIPTION
**Pull request by Hypercubed**

### CODE REVIEW

File: src/components/input/input.component.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@ $input-icon-color: $color-grey-300;
 
   .ngx-input-wrap {
     position: relative;
-    display: inline-block;
+    display: block;
```
Comment (amcdnl) [inline src/components/input/input.component.scss]: inputs needs to be able to sit beside each other.

File: demo/app/app.template.html [hunk 2]
```diff
@@ -372,7 +558,7 @@ <h3 class="style-header">Align Bottom</h3>
             </ngx-select>
           </td>
         </tr>
-      </table>
+      </table -->
```
Comment (amcdnl) [inline demo/app/app.template.html]: Why'd you comment this out?

Comment (Hypercubed) [inline demo/app/app.template.html]: Should be removed, only baseline alignment is relevant.

File: src/components/input/input.component.scss (hunk 1 above)
Comment (Hypercubed) [inline src/components/input/input.component.scss]: This is the `.ngx-input-wrap` class.  This should not effect the ngx-input elements, which were and still are, block elements.

**Comment by amcdnl** [comment]
Rebase and go ahead and merge @Hypercubed

**Pull request merged**